content/en/dashboards/functions/arithmetic.md

```diff
@@ -57,6 +57,12 @@ If a metric, `const_1{*}`, is a constant with the value of `1`, then `cumsum(con
 
 {{< img src="dashboards/functions/arithmetic/cumsum.png" alt="cum sum function with abs" style="width:80%;">}}
 
+## Cumulative sum in monitors
+
+Cumulative sum should be avoided in monitor queries, because the cumulative sum function is a visual function. When using within a dashboard or notebook, the points will reflect values based on the timeframe that is selected. This doesn’t translate well within a monitor as the monitor doesn’t have a sense of which timeframe to use.
```

Comment:
```suggestion
Cumulative sum should be avoided in monitor queries, because the cumulative sum function is a visual function. When used in a dashboard or notebook, the points will reflect values based on the selected timeframe. This doesn’t translate well in a monitor as the monitor doesn’t have a sense of which timeframe to use.
```